pkg/machinery/marker_test.go

```diff
@@ -39,8 +39,30 @@ var _ = Describe("Marker", func() {
 	Context("String", func() {
 		DescribeTable("should return the right string representation",
 			func(marker Marker, str string) { Expect(marker.String()).To(Equal(str)) },
-			Entry("for go files", Marker{comment: "//", value: "test"}, "// +kubebuilder:scaffold:test"),
-			Entry("for yaml files", Marker{comment: "#", value: "test"}, "# +kubebuilder:scaffold:test"),
+			Entry("for go files", Marker{prefix: kbPrefix, comment: "//", value: "test"}, "// +kubebuilder:scaffold:test"),
+			Entry("for yaml files", Marker{prefix: kbPrefix, comment: "#", value: "test"}, "# +kubebuilder:scaffold:test"),
+		)
+	})
+})
+
+var _ = Describe("NewMarkerFor", func() {
+	Context("String", func() {
+		DescribeTable("should return the right string representation",
+			func(marker Marker, str string) { Expect(marker.String()).To(Equal(str)) },
+			Entry("for yaml files", NewMarkerFor("test.yaml", "test"), "# +kubebuilder:scaffold:test"),
+		)
+	})
+})
+
+var _ = Describe("NewMarkerWithPrefixFor", func() {
+	Context("String", func() {
+		DescribeTable("should return the right string representation",
+			func(marker Marker, str string) { Expect(marker.String()).To(Equal(str)) },
+			Entry("for yaml files", NewMarkerWithPrefixFor("custom:scaffold", "test.yaml", "test"), "# +custom:scaffold:test"),
+			Entry("for yaml files", NewMarkerWithPrefixFor("+custom:scaffold", "test.yaml", "test"), "# +custom:scaffold:test"),
+			Entry("for yaml files", NewMarkerWithPrefixFor("custom:scaffold:", "test.yaml", "test"), "# +custom:scaffold:test"),
+			Entry("for yaml files", NewMarkerWithPrefixFor("+custom:scaffold:", "test.yaml", "test"), "# +custom:scaffold:test"),
+			Entry("for yaml files", NewMarkerWithPrefixFor(" +custom:scaffold: ", "test.yaml", "test"), "# +custom:scaffold:test"),
```

Comment:
It is not passing in the linter due the sieze of the line,
 Could you please break the line?
 
 > pkg/machinery/marker_test.go:65: line is 122 characters (lll)
  			Entry("for yaml files", NewMarkerWithPrefixFor(" +custom:scaffold: ", "test.yaml", "test"), "#

Reply:
I fixed the linter issues.